### PR TITLE
🚸(github) adding github issue and PR templates

### PR DESCRIPTION
**Pull request by sampaccoud**
## Purpose

We want to have more formalism in the way PR and Issues are reported.

## Proposal

Let's add an `ISSUE_TEMPLATE.md` file and a `PULL_REQUEST_TEMPLATE.md` file in a `.github` folder so that Github automatically proposes these templates when someone is opening an issue (resp. pull request).

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+Description...
+
+
+## Actual Behavior
+
+Description...
+
+
+## Steps to Reproduce the Problem
```
Comment (jmaupetit) [inline .github/ISSUE_TEMPLATE.md]: An issue is not always a problem :wink:

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+1. item 1...
+2. item 2...
+
+
+## Specifications
+
+- Version: ...
+- Platform: ...
+- Subsystem: ...
```
Comment (jmaupetit) [inline .github/ISSUE_TEMPLATE.md]: What is a subsystem?

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (sampaccoud) [reply .github/ISSUE_TEMPLATE.md]: I propose: `steps to reproduce"

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (sampaccoud) [reply .github/ISSUE_TEMPLATE.md]: good question... Let me remove it.

**Pull request merged**